### PR TITLE
SSR-3217 Migrate docusaurus Adding v1.4.0

### PR DESCRIPTION
**Pull request by nmanhit**

### CODE REVIEW

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+</section>
+```
+
+## JavaScript/CSS カスタマイズ（設定画面）
+
+設定画面の実装コードを解説します。<br/>
+以下の様な JavaScript ファイル（config.js）を js フォルダに配置します。<br/>
+
+
+```javascript title="config.js"
```
Comment (dong0) [inline docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:137]: You changed "\*\*\*config.js***" to "title="config.js"".
It's different from v1.11.0.
Please check it.

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+// When the Cancel button in Dialog is clicked
+dialogCancelButton.addEventListener("click", event => {
+  dialog.close();
+});
+```
+
+## JavaScript/CSS カスタマイズ（アプリ画面）
+
+アプリ画面の実装コードを解説します。<br/>
+以下のようなJavaScript ファイル（desktop.js）を js フォルダに配置します。<br/>
```
Comment (dong0) [inline docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:372]:
```suggestion
以下の様なJavaScript ファイル（desktop.js）を js フォルダに配置します。<br/>
```

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+プラグイン設定画面のコンポーネントから値を引き継いで、アプリのカスタマイズで利用しているのがポイントです。<br/>
+DatePicker コンポーネントの value が yyyy-mm-dd 形式のため、
+比較しやすいように本日の日付も yyyy-mm-dd 形式で取得しています。<br/>
+今回は luxon.js という外部の日付ライブラリを利用しています。
+
+
+```javascript title="desktop.js"
```
Comment (dong0) [inline docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:379]: You changed "\*\*\*desktop.js***" to "title="desktop.js"".
It's different from v1.11.0.
Please check it.

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+```
+
+今回はプラグイン設定画面のカスタマイズで kintone UI Component を使用するので、config オブジェクトの中に UMD ファイルのパスを記載します。<br/>
+UMD ファイルの導入方法は、[Quick Start](../getting-started/quick-start.md) をご覧ください。
+
+
+## HTMLカスタマイズ
+
+プラグイン設定画面用の HTML ファイルを作成します。<br/>
+今回は以下の様に作成します。div 要素に設定した id を利用して、kintone UI Component の各パーツを配置していきます。
```
Comment (dong0) [inline docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:103]:
```suggestion
今回は以下のように作成します。div 要素に設定した id を利用して、kintone UI Component の各パーツを配置していきます。
```

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+    <p class="kintoneplugin-row">
+      <div id="button_space"></div>
+    </p>
+</section>
+```
+
+## JavaScript/CSS カスタマイズ（設定画面）
+
+設定画面の実装コードを解説します。<br/>
+以下の様な JavaScript ファイル（config.js）を js フォルダに配置します。<br/>
```
Comment (dong0) [inline docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:134]:
```suggestion
以下のような JavaScript ファイル（config.js）を js フォルダに配置します。<br/>
```

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md (hunk 4 above)
Comment (nmanhit) [reply docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:103]: This paragraph above is used in version 1.0.4 and only gets chnaged from version 1.10.0.
Please double-check
https://kintone-ui-component.netlify.app/docs/ja/1.4.0/guides/format-setting-plugin
https://kintone-ui-component.netlify.app/docs/ja/1.11.0/guides/format-setting-plugin

Comment (dong0) [reply docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:103]: Reply: https://github.com/kintone-labs/kintone-ui-component/pull/1397#discussion_r1278873998

Comment (TomokoMiyake) [reply docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md:103]: Reply: https://github.com/kintone-labs/kintone-ui-component/pull/1397#discussion_r1279265605

**Review by TomokoMiyake — APPROVED** [review]
I checked the changed part of v1.4.0 following the release notes and confirmed it.
LGTM except for Dong-san's review point.
https://github.com/kintone-labs/kintone-ui-component/releases/tag/v1.4.0

**Pull request merged**